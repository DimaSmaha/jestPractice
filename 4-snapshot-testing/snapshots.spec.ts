const user = {
  name: "Tony Tinkerton",
  age: 42,
  job: "inventor",
};

describe("Snaps", () => {
  test("without snapshots", () => {
    const userString = '{"name":"Tony Tinkerton","age":42,"job":"inventor"}';

    expect(JSON.stringify(user)).toEqual(userString);
  });

  test("user matches with snapshot", () => {
    expect(user).toMatchSnapshot();
  });
});
