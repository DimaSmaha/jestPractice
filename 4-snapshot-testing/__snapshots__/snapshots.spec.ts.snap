// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snaps user matches with snapshot 1`] = `
{
  "age": 42,
  "job": "inventor",
  "name": "Tony Tinkerton",
}
`;
